src/os/unix/unix.cpp

```diff
@@ -266,7 +270,17 @@ int CDECL main(int argc, char *argv[])
 #ifndef WITH_COCOA
 bool GetClipboardContents(char *buffer, const char *last)
 {
+#ifdef WITH_SDL2
+	char *clip;
+
+	if (SDL_HasClipboardText() == SDL_TRUE && (clip = SDL_GetClipboardText()) != NULL) {
+		strecpy(buffer, clip, last);
+		return true;
+	}
+	else return false;
```

Comment:
The else clause needs braces.

I'd prefer this structured with a series of early "false" returns, also to avoid declaring `clip` without initialisation.